includes/sanitizers/class-amp-link-sanitizer.php

```diff
@@ -295,6 +304,11 @@ public function is_frontend_url( $url ) {
 			return false;
 		}
 
+		// Skip adding query var to links on other paths.
+		if ( ! empty( $parsed_url['path'] ) && false !== strpos( $parsed_url['path'], $this->home_path ) ) {
```

Comment:
This doesn't seem quite right. We need to check if the parsed path _begins_ with the home path, not that it exists anywhere in the string. Consider case where WP is installed at `/blog/` but the URL being linked to `/about/my/blog/`.

```suggestion
		if ( ! empty( $parsed_url['path'] ) && 0 !== strpos( $parsed_url['path'], $this->home_path ) ) {
```

Reply:
Ahh. got it. Thanks for catching it 👍🏼 .